react-hello-world-master/src/Dashboard.test.js

```diff
@@ -2,8 +2,41 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Dashboard from './dashboard';
 
+class messages{
```

Comment:
```git rebase origin/main```

Then import messages and channels from the models.js, because in your current history there is no such file